app/assets/javascripts/collections/user.js

```diff
@@ -0,0 +1,14 @@
+
+(function(App) {
+
+  'use strict';
+
+  App.Collection.User = App.Collection.Table.extend({
+
+    // url: 'admin/users.json'
+
+    url: '/users.json'
```

Comment:
you can:
a) add it yourself
b) ask a back dev to add it for you

you are currently binding your implementation to a json structure that does not reflect the current structure of the API we have.